lib/govukFrontendPaths.js

```diff
@@ -26,7 +26,7 @@ function govukFrontendPaths (searchPaths = []) {
     baseDir,
 
     includePath: `/${path.relative(baseDir, includeDir)}`,
-    assetPath: `/${path.relative(baseDir, path.join(includeDir, 'assets'))}`,
+    assetPath: `/${path.relative(baseDir, path.join(includeDir, 'assets/rebrand'))}`,
```

Comment:
I think this'll affect both the 'manage prototype' section and prototypes, and break assets entirely for older versions of GOV.UK Frontend.

Would be nice to fix #2427 at some point but to get this merged perhaps we could just append `rebrand` when we set `assetPath` in `lib/nunjucks/views/manage-prototype/layout.njk`?

Reply:
So I've tried to resolve both. Firstly I've tied the assets fragment to if the app config has the rebrand options in it and are set to true. Secondly I've added a check to the management layout template that appends `rebrand` if `assets/rebrand` isn't present in `assetPath`. Let me know what you reckon.

Reply:
Just to check, which of these are we aiming for:

1. The management pages **always** use the updated brand
2. The management pages use the rebrand **only if the prototype itself** uses the updated brand

I assumed we were aiming for 1 when I reviewed which might have confused things.

Reply:
1 is what we're aiming for. Do you reckon that's not what we're gonna get? From local testing the current solution lands at 1.

Reply:
Before the last force-push you were setting the rebrand global based on the plugin config in `lib/manage-prototype-handlers.js` – I saw that just wanted to double check we didn't have our wires crossed 👍🏻